Changelog.md

```diff
@@ -17,10 +17,10 @@
 
 -   Blocking dialog when disconnecting a device when the Programming dialog is
     open.
--   No longer auto reselect to the least disconnected device if is is in the
-    device list when clicking the `Auto Reconnect` Toggle.
+-   No longer auto reselect to the least disconnected device if is in the device
```

Comment:
```suggestion
-   No longer auto reselect to the last disconnected device if it is in the device
```

Reply:
Thanks, I did not go through previous versions changelog a lot :)